### DISCUSSION TITLE
search for term IDs by Xrefs

### DISCUSSION THREAD

**Discussion by Antonialock** [geneontology/helpdesk#355] [Q&A] [answered]
**Please describe your question, idea, or concern.**

Hello,
is there a way in amigo (or quickgo for that matter) to search for terms by cross-references? e.g. EC or RHEA IDs?
Cheers

**Comment by pgaudet** [comment, marked as answer]
Not that I know of - I use protégé

The RHEA site allows searching for GO and EC terms, and displays GO term mappings; maybe this works for you?

**Comment by Antonialock** [comment]
ah thanks! That's a good tip
